contents/blog/using-vc-carefully.md

```diff
@@ -0,0 +1,117 @@
+
+There were three surprising lessons we had during this phase:
+
+1 - charging customers for a product led to more usage. Early paying customers were more demanding of quality. When we fixed quality issues, our free _and_ paid products both accelerated
+2 - how important it is to track your anti-Ideal Customer Profile. Don't just monitor deals that close, but track those that are dragging on - what do they have in common?
+3 - how quickly we went from getting the first 5 paying customers, to 100.
+
+## We have Product Market Fit, time to raise more money?
+
+Many companies do this. The reason is pretty simple - if you grow by hiring salespeople, you can use venture capital to hire faster than you could otherwise afford.
+
+We took the stance that we should get to $10M in recurring revenue, and then could go nuts. This would mean we could raise capital easily, on great terms, and it'd mean we had very solid product market fit.
+
+However, now that figure is getting close - we're no-longer convinced this is the right thing to do. The way we grow is pretty simple:
+
+* Make the existing products we have work better
+* Add extra products to the platform
+* Content marketing (👋)
+
+Normal companies probably _should_ raise venture capital, since most grow by hiring a big sales team whilst the product tries to keep up. PostHog's main growth lever is shipping.
+
+## Developer math
+
+So, if we grow by shipping, and we want to grow as fast as possible - we need to be able to ship as fast as possible.
+
+Adding developers is an intense process for us - particularly as we have found we work best with experienced people. We think it's the nature of an all remote company with a wide product - we need people that thrive in autonomy, and those folk normally don't need as much coaching.
+
+So, if we add lots of people, will we ship faster?
+
+To a certain extent.
+
+* Every hire involves dozens of interviews
+* New technical hires require onboarding
+
+Since our revenue is growing fast, we could triple or more our team next year, just from revenue growth. Hiring more than that feels like our developers won't really be writing code. So, we don't think a huge fundraise is the right thing to do.
+
+## So what should you take from this?
+
+For us, I think we probably will fundraise again. However, our goals may be different to the norm - like just making sure we have more working capital so we can stay long term focused, to provide employee secondaries, or to get the ideal board member into the company.
+
+Venture Capital is a tool, not a religion. Don't blindly follow what you hear others are doing on TechCrunch.
+
+Instead, have a clear fundraising purpose. If it's to accelerate growth, great. Just ask yourself if it will _actually_ help achieve that - based on how _your company_ is growing.
```

Comment:
```suggestion
Instead, have a clear fundraising purpose. If it's to accelerate growth, great. Just ask yourself if it will _actually_ help achieve that based on how _your company_ is growing.
```